bindings/csharp/http/README.md

```diff
@@ -39,8 +39,8 @@ name: Install C# dependencies
 
 ```bash
 cd ./batch
-ls
 dotnet restore
+dotnet build
```

Comment:
Not necessary, with dotnet run below.  I'm trying to keep each language with their logical restore step (npm install, pip3 install, etc) here for consistency.  
```suggestion
```